scripts/import-github-tokens.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+const { promises: fs } = require('fs')
+const Redis = require('ioredis')
+
+const key = 'githubUserTokens'
+
+async function loadTokens() {
+  const contents = await fs.readFile('all_tokens_uniq.json', 'utf8')
+  const tokens = JSON.parse(contents)
+  console.log(`${tokens.length} tokens loaded`)
+  return tokens
+}
+
+function createClient() {
+  const redis = new Redis(process.env.REDIS_URL, {
+    tls: { servername: new URL(process.env.REDIS_URL).hostname },
```

Comment:
I take it this script is only intended to be used with compose, since it doesn't have the sample conditional `rediss` check?

Reply:
Yea, correct… also I wanted to be super sure I was using TLS ;)

The script is also sort of throwaway, but keeping-around-for-reference.